### PR TITLE
Jdp210201 2

### PR DESCRIPTION
**Pull request by lukkra1989**
JDP210201-2: I've created GroupController with temporary classes GroupDto and Group.

### CODE REVIEW

File: src/main/java/com/kodilla/ecommercee/GenericEntityRepository.java [hunk 1]
```diff
@@ -4,4 +4,5 @@
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface GenericEntityRepository extends JpaRepository<GenericEntity, Long> { }
```
Comment (Ktubbo) [inline src/main/java/com/kodilla/ecommercee/GenericEntityRepository.java:7]: I think that it shouldn't be changed.

File: src/main/java/com/kodilla/ecommercee/GroupDto.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package com.kodilla.ecommercee;
+
+public class GroupDto {
```
Comment (Ktubbo) [inline src/main/java/com/kodilla/ecommercee/GroupDto.java]: I don't know why there is not used lombok annotations (@ Getter, @ Setter), when you are using it in other classes.

File: src/main/java/com/kodilla/ecommercee/GenericEntityRepository.java (hunk 1 above)
Comment (Ktubbo) [inline src/main/java/com/kodilla/ecommercee/GenericEntityRepository.java]: I think that it shouldn't be changed.

File: src/main/java/com/kodilla/ecommercee/GroupController.java [hunk 3]
```diff
@@ -1,4 +1,33 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
```
Comment (Ktubbo) [inline src/main/java/com/kodilla/ecommercee/GroupController.java]: Those two imports are not needed.

**Review by Ktubbo — APPROVED** [review]

**Review by Bauaser-kun — APPROVED** [review]

**Pull request merged**